### ISSUE TITLE
Compatibility errors between Diart requirements and Pyannote

### ISSUE THREAD

**Issue by Farissoliman** [juanmc2005/diart#160]
After following the instructions on the `README` file, when I run `pip install diart`, I get this error message:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. 
This behaviour is the source of the following dependency conflicts.
pyannote-audio 2.1.1 requires torchaudio>=2.0.0, but you have torchaudio 0.13.1 which is incompatible.
```
I then try running `pip install git+https://github.com/pyannote/pyannote-audio.git@develop#egg=pyannote-audio` to get the latest form of pyannote-audio which includes torchaudio-2.0.2, but then I get this error:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. 
This behaviour is the source of the following dependency conflicts.
diart 0.5.1 requires torchaudio<1.0,>=0.10, but you have torchaudio 2.0.2 which is incompatible.
```
Is there any fix or merge that I can do to fix these issues?

**Comment by juanmc2005** [comment]
Hi @Farissoliman, currently the installation is a bit clunky. This is also related to #158.
This is something on which I'm going to put special focus on in the coming weeks. Currently I'm just lacking the time to address it because diart's not my main job.

For now you can ignore this warning and use the newest torchaudio version, which should work well. Also, make sure you install the latest version v0.7, you're currently installing v0.5.1.

If you want to contribute, adding an `environment.yml` file with the latest working packages would be an amazing start.

**Comment by juanmc2005** [comment]
Fix implemented in #181. It will appear in next release (v0.8)
